sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridPublisherClient.cs

```diff
@@ -212,22 +216,11 @@ private async Task<Response> PublishCloudEventsInternal(IEnumerable<CloudEvent>
                     // The 'Data' property is optional for CloudEvents
```

Comment:
nit: Not only is Data optional, but passing binary data into the ctor would result in this being null.